configs/setups/awiesm/awiesm.yaml

```diff
@@ -325,20 +335,29 @@ fesom:
         # --- icebergs ---
         with_icb: ${general.with_icb}
         # --- icebergs ---
+                        
+        outdata_targets:
+                fesom: "*.fesom.*.nc"
```

Comment:
This will most likely break older versions of this setup. I'll come back to you later on with a suggestion for a solution, if there is really a problem.

Reply:
How so?

The monthly files break pyfesom2 and tripyview scripts, which are basically the only way to analyse FESOM2 output for most users.

Reply:
I would argue this should be an impulse to Patrick to make the tripyview tools more flexible. It should not rely on hard-code file patterns...

I see no difficulties in this (rather aggressive) glob, though.

Reply:
> How so?
> 
> The monthly files break pyfesom2 and tripyview scripts, which are basically the only way to analyse FESOM2 output for most users.

Many of our old simulations have a different name convention. That means restarting from them with this new change will not be out of the box, but you'll need to change somethings in the runscript or rename the files. But on a second thought, if we want this change for the future we have to eventually make it happen, so let's go for it I'd say. We will assist people who cannot branch off from old simulations.